### PR TITLE
Add STUDIO_SHORT_NAME

### PR DESCRIPTION
**Pull request by ovnicraft**
Added short name var for studio, this helps to redefined from servers-vars.yml

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ovnicraft! I've created [OSPR-453](https://openedx.atlassian.net/browse/OSPR-453) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams. 

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ('this must be merged by XX date', and why that is)
- partner information ('this is a course on edx.org')
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the Github pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

We can't start reviewing your pull request until you've added yourself to the [AUTHORS](https://github.com/edx/configuration/blob/master/AUTHORS) file. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst) file for more information.

**Comment by sarina** [comment]
@ovnicraft can you please also add yourself to AUTHORS? You should do that for each new repo you contribute to!

**Comment by sarina** [comment]
@feanil this is a configuration change needed due to changes that @singingwolfboy recently made to Studio settings.

ref #1771 and https://github.com/edx/edx-platform/pull/6468

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -105,6 +105,7 @@ EDXAPP_VIDEO_CDN_URLS:
 
 EDXAPP_PLATFORM_NAME: 'Your Platform Name Here'
 EDXAPP_STUDIO_NAME: 'Studio'
+EDXAPP_STUDIO_SHORT_NAME: 'Studio'
```
Comment (sarina) [inline playbooks/roles/edxapp/defaults/main.yml:108]: @ovnicraft don't you need to add something in the lower section too? See https://github.com/edx/configuration/pull/1771/files#diff-9bffb77ca37dcc0aead699b822b29095R597

**Comment by ovnicraft** [comment]
@sarina yes, done !

**Comment by feanil** [comment]
@sarina @singingwolfboy does the original change mean that we need to set a default of 'edX Studio' in our environments so that we don't end up using the default values?

**Comment by sarina** [comment]
STUDIO_SHORT_NAME should be "Studio" on our servers. I was under the impression that we were definitely using STUDIO_NAME to mean "edX Studio" --- but @singingwolfboy as the author of the change would know for certain.

**Comment by singingwolfboy** [comment]
That's correct. On the production configuration for edx.org, `STUDIO_NAME` should be set to "edX Studio", and `STUDIO_SHORT_NAME` should be set to "Studio".

**Comment by feanil** [comment]
:+1:

**Pull request merged**